packages/asc-core/src/components/ContextMenu/ContextMenuItem.ts

```diff
@@ -0,0 +1,31 @@
+import styled from '../../styled-components'
+import { ContextMenu } from './types'
+import Icon from '../Icon'
+import { getColor, getTypography } from '../../utils/themeUtils'
+
+const ContextMenuItem = styled.li<ContextMenu.ContextMenuItemProps>`
+  padding: 0;
+  margin: 0;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  height: 36px;
+  width: 100%;
+  cursor: pointer;
+  font-size: ${({ theme }) => getTypography(theme).fontWeightRegular};
+
+  ${Icon} {
+    margin: 5px 12px;
```

Comment:
The design system mentions having a horizontal spacing of `5px` instead of `12px` for icons in link lists. The items in the list should have a vertical spacing of `12px`, though.